CHANGES.md

```diff
@@ -1,3 +1,14 @@
+## edge-19.8.4
+
+This edge release is a release candidate for `stable-2.5`.
+
+* CLI
+  * Introduced `--use-wait-flag` to the `linkerd install-cni` command, to
+    configure the CNI plugin to use the `-w` flag for `iptables` commands
+* Controller
+  * Disabled the tap gRPC server listener. All tap requests now require RBAC
+    authentication and authorization
```

Comment:
#3194 got missed in the last edge, so you might wanna add it here. It's about supporting the proxy config override annotations at the namespace level.

Reply:
Good catch. I'm going to add this under `edge-19.8.3` since that's when the feature was introduced.